e-learning-webapps/src/main/webapp/vue-app/js/tutoService.js

```diff
@@ -0,0 +1,40 @@
+export function getTutos() {
+  return fetch('/portal/rest/tuto/getAllTutos', {
+    method: 'GET'
+  })
+    .then((response) => response.json());

```

Comment:
Add error message in case of server error
.then(resp => {
    if (!response || !response.ok) {
      throw new Error('Response code indicates a server error', response);
    } else {
      return response.json();
    }
  });